bindings/core/src/method_handler/client.rs

```diff
@@ -176,7 +176,10 @@ pub(crate) async fn call_client_method_internal(client: &Client, method: ClientM
                 .get_addresses(&secret_manager)
                 .set_options(options)?
                 .finish()
-                .await?;
+                .await?
+                .into_iter()
+                .map(|a| a.to_string())
+                .collect();
             Response::GeneratedAddresses(addresses)
```

Comment:
Do we not want this Response to be a vec of Bech32Address ?

Reply:
Missed it, thanks 👍